src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -10,30 +10,41 @@
 
 /**
  * Finds and lists all persons in address book whose name contains any of the argument keywords.
- * Keyword matching is case sensitive.
+ * Keyword matching is case insensitive.
  */
 public class FindCommand extends Command {
 
     public static final String COMMAND_WORD = "find";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
-            + "the specified keywords (case-sensitive) and displays them as a list with index numbers.\n"
+            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
             + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
             + "Example: " + COMMAND_WORD + " alice bob charlie";
 
     private final Set<String> keywords;
+    private final Set<String> normalKeywords;
 
     public FindCommand(Set<String> keywords) {
-        this.keywords = keywords;
+        this.keywords = new HashSet<>();
+        for (String word: keywords){
+            this.keywords.add(word.toUpperCase());
+        }
+        this.normalKeywords = keywords;
     }
 
     /**
      * Returns a copy of keywords in this command.
      */
-    public Set<String> getKeywords() {
+    public Set<String> getKeywords(){
```

Comment:
Need a space between `) {`.
You made the correct one incorrect :stuck_out_tongue: